pages/platform/ai/deploy_tuto_11_rasa_chatbot_flask/guide.en-gb.md

```diff
@@ -0,0 +1,262 @@
+
+In a previous tutorial, we created and trained a Rasa Chatbot with AI Notebooks : [How to create and train a chatbot on OVHcloud](https://docs.ovh.com/gb/en/publiccloud/ai/).
+Now, the aim of this tutorial is to deploy a chatbot with OVHcloud AI Tools. We also train our chatbot with AI Training. 
+
+We used the famous open source framework [Rasa](https://rasa.community/) to build the chatbot. 
+To deploy our chatbot, we will use the [Flask framework](https://flask.palletsprojects.com/en/2.2.x/) to create a web application frontend. 
+
+This tutorial objectives are:
+1. Secure our Flask application 
+2. Deploy our rasa model with AI deploy
+3. Deploy our flask application frontend and speak with the chatbot. 
+
+Here is the small schema to explain to you how it works :
+
+![image](images/diagramme.png){.thumbnail}
+
+**Requirements**
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
+- A Public Cloud project created.
+- The ovhai CLI interface installed on your laptop. More information [here](https://docs.ovh.com/gb/en/publiccloud/ai/cli/install-client/).
+- [Docker](https://www.docker.com/get-started) installed on your local computer
+- Make sure you have a Docker Hub [account](https://hub.docker.com/) or another Docker registry.
+- Some knowledge about building image and [Dockerfile](https://docs.docker.com/engine/reference/builder/).
+
+## Instructions
+
+We will create two AI apps to deploy the rasa model. First, you will have to create two environments variables for the flask app. Let's do this !
+
+### Clone our example repository 
+
+Please make sure you have cloned the repository git. 
+You can find it here : [repo git](https://github.com/Victor2103/ai-training-examples). 
+
+### Create environment variables
+
+Frontend and Backend (the chatbot) have to communicate safely and securely. We will generate security keys for that.
+
+The first variable will be the secret key for the Json web token signature to access your Rasa chatbot. To generate this key, I've used python. If you have python2.6+ installed on your machine, you can run python inside a terminal and then : 
+
+```python
+import secrets
+print(secrets.token_urlsafe())
+# Should display something like this : 
+# dux3BudMxlRSm1GI3IoBEuS7UWVU3nYGJ9l_0Cd3rms
+```
+
+The second one will be the algorithm used for the json web token. The algorithm will use will be H256.
+
+You have your two environnements variables. Time to save it ! create a '.env' file inside the folder `flask_app`. Your .env should look like this : 
+
+```
+JWT_SECRET_KEY=your-jwebtoken-generated-before
+JWT_ALGORITHM=HS256
+```
+
+Your environment variables are saved. One more thing to do is to add in the docker-compose.yml file in the image Rasa the JWT_SECRET_KEY value. The value must be the same in the `.env` file. Otherwise, your model will not be able to run. Now, let's test locally our app or let's deploy our chatbot !
```

Comment:
```suggestion
Your environment variables are saved. One more thing to do is to add in the `docker-compose.yml` file in the image Rasa the `JWT_SECRET_KEY` value. The value must be the same in the `.env` file. Otherwise, your model will not be able to run. Now, let's test locally our app or let's deploy our chatbot!
```